lib/matplotlib/backends/backend_pdf.py

```diff
@@ -178,18 +178,16 @@ def pdfRepr(obj):
     # anything, so the caller must ensure that PDF names are
     # represented as Name objects.
     elif isinstance(obj, dict):
-        r = [b"<<"]
-        r.extend([Name(key).pdfRepr() + b" " + pdfRepr(obj[key])
-                  for key in sorted(obj)])
-        r.append(b">>")
-        return fill(r)
+        return fill([
```

Comment:
I think this would better be formatted as
~~~
        repr = [Name(key).pdfRepr() + b" " + pdfRepr(obj[key])
                for key in sorted(obj)]
        return fill([b"<<", *repr, b">>"])
~~~